src/demo-app/DemoApp.tsx

```diff
@@ -18,7 +17,7 @@ function DemoApp(): JSX.Element {
     <Router>
       <Switch>
         <Route path="/mock">
-          <Suspense fallback={<Loader />}>
+          <Suspense fallback={<></>}>
```

Comment:
Showing a loader while fetching the metadata looked weird. Given how fast the fetch is, I think a blank page is cleaner.